Resources/Prototypes/_EinsteinEngines/Species/ipc.yml

```
@@ -202,7 +233,9 @@
 - type: marking
   id: MobIPCLHandDefault
   bodyPart: LHand
-  markingCategory: Arms
+  markingCategory: LeftHand
+  speciesRestriction: [ IPC ]
+  markingCategory: RightArm
```

YAMLlint (1.37.1)

[error] duplication of key "markingCategory" in mapping

(key-duplicates)